php/ng/map.jinja

```diff
@@ -2272,6 +2272,8 @@
                             'conf': confdir + '/fpm/php-fpm.conf',
                             'ini': confdir + '/fpm/php.ini',
                             'pools': confdir +  '/fpm/pool.d',
+                            'user': 'root',
+                            'group': 'root',
                             'service': 'php' + phpng_version + '-fpm',
                             'user': 'root',
                             'group': 'root',
```

Comment:
@philpep I've revisited this PR as I'm rebasing #183 and I've noticed that the two additions from this PR is already available here (lines 2278 and 2279).  Is there are reason why that didn't already work for you?